src/main/java/com/walking/intensive/chapter2/task10/Task10.java

```diff
@@ -11,11 +11,24 @@
  */
 public class Task10 {
     public static void main(String[] args) {
-//        Для собственных проверок можете делать любые изменения в этом методе
+
     }
 
     static boolean isPalindrome(String inputString) {
-        // Ваш код
-        return false;
+
+        if (inputString == null || inputString.length() <= 1) {
+            return false;
+        }
+
+       String ignoreChars = " .,:;!?\"()—-";
+       StringBuilder builder = new StringBuilder();
+
+       for (int i=0; i < inputString.length(); i++) {
```

Comment:
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-2024/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter2/task10/Task10.java:26:18: '=' is not followed by whitespace. [WhitespaceAround]

> Task :checkstyleMain
Error: eckstyle] [ERROR] /home/runner/work/intensive-tasks-20[24](https://github.com/KFalcon2022/intensive-tasks-2024/actions/runs/12044416295/job/33633933790?pr=438#step:4:25)/intensive-tasks-2024/src/main/java/com/walking/intensive/chapter2/task10/Task10.java:[26](https://github.com/KFalcon2022/intensive-tasks-2024/actions/runs/12044416295/job/33633933790?pr=438#step:4:27):18: '=' is not preceded with whitespace. [WhitespaceAround]